deny.toml

```diff
@@ -23,6 +23,8 @@ allow = [
 ]
 
 exceptions = [
+    # Explicitly allows MPL-2 being pulled in through reqwest's and actix's rustls dependency chain (which uses webpki)
+    { name = "webpki-roots", allow = ["MPL-2.0"], version = "*" },
```

Comment:
Sorry about the extra work. :/

Reply:
Oh no worries! We had no way of knowing this was going to end up being a dependency again since it was removed by the AWS smithy libraries 😄